imports/plugins/core/checkout/client/templates/checkout/addressBook/addressBook.js

```diff
@@ -0,0 +1,28 @@
+import { Template } from "meteor/templating";
+import { Components } from "@reactioncommerce/reaction-components";
+
+/**
+ * Helpers: Checkout Address Book
+ */
+Template.checkoutAddressBook.helpers({
+  /**
+   * @method AddressBook
+   * @summary returns a component for updating a user's address.
+   * @since 2.0.0
+   * @return {Object} - contains the component for updating a user's address.
+   */
+  AddressBook() {
+    const { status, position } = Template.instance().data;
```

Comment:
`template.data` is the nonreactive one. Use `Template.currentData()` instead to ensure it will update when `status` or `position` change after initial render.

```js
const { status, position } = Template.currentData();
```